### PR TITLE
README reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -3,8 +3,8 @@
    :target: https://weblate.org/
    :height: 80px
 
-**Weblate is a copylefted libre software web-based continuous localization system,
-used by over 2500 libre projects and companies in more than 165 countries.**
+Weblate is a copylefted libre software web-based continuous localization system, \
+used by over 2500 libre projects and companies in more than 165 countries.
```
[the remark this answers is not in the document]
Comment (comradekingu) [reply README.rst:7]: It isn't in bold on https://weblate.org/en/, (only continuous is) and it isn't "copylefted" on https://hosted.weblate.org/about/.
My take is that non-bold is more elegant.

File: README.rst [hunk 2]
```diff
@@ -17,9 +17,17 @@ Hosted Weblate customizations
...
    :target: https://codecov.io/gh/WeblateOrg/hosted
 
-These customizations are used on the `Hosted Weblate service
-<https://weblate.org/hosting/>`_.
+These customizations are used on the `Hosted Weblate service <https://weblate.org/hosting/>`_.
 
-You can use them as an example how to customize `Weblate
-<https://weblate.org/>`_, but there are probably not generally usable.  If you
-think something should be part of Weblate, please open an issue.
+Hosted Weblate, error pages, legal
```
[the remark this answers is not in the document]
Comment (comradekingu) [reply README.rst:22]: Can't remember what I was thinking

File: README.rst [hunk 3]
```diff
@@ -17,9 +17,17 @@ Hosted Weblate customizations
...
+
+.. image:: https://hosted.weblate.org/widgets/weblate/-/hosted/svg-badge.svg
+   :target: https://hosted.weblate.org/engage/weblate/
+.. image:: https://hosted.weblate.org/widgets/weblate/-/hosted-error-pages/svg-badge.svg
+   :target: https://hosted.weblate.org/engage/weblate/
+.. image:: https://hosted.weblate.org/widgets/weblate/-/legal/svg-badge.svg
+   :target: https://hosted.weblate.org/engage/weblate/
+
+This first-party instance serves as a possible example of how to customize `Weblate <https://weblate.org/>`_, \
+but it is not made to adaptable for general use. \
```
[the remark this answers is not in the document]
Comment (comradekingu) [reply README.rst:32]: Usually it line-shifts, but the preview doesn't think so.

File: README.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (nijel) [reply README.rst:7]: It's bold to distinguish it from the actual project description. Maybe using italic would be more appropriate here, but I still think it should be visually different than rest of the text.

**Pull request closed without merging**